lang/tests/src/test/scala/com/wavesplatform/lang/IntegrationTest.scala

```diff
@@ -1380,25 +1380,25 @@ class IntegrationTest extends PropSpec with PropertyChecks with ScriptGen with M
   }
 
   property("toBase16String limit 8Kb from V4") {
-    val base16String8Kb = "FEDCBA9876543210" * 1024
+    val base16String8Kb = "fedcba9876543210" * 1024
```

Comment:
what if `"FEDCBA9876543210"`?
Should there be an error? Or should a function successfully handle uppercase letters?
Add a test.

Reply:
ok